packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -66,26 +66,27 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     timeFormat: input.timeFormat,
   };
 
-  if (
-    input.slug &&
-    IS_TEAM_BILLING_ENABLED &&
-    /** If the team doesn't have a slug we can assume that it hasn't been published yet. */
-    !prevOrganisation.slug
-  ) {
-    // Save it on the metadata so we can use it later
-    data.metadata = {
-      requestedSlug: input.slug,
-    };
-  } else {
-    data.slug = input.slug;
-
-    // If we save slug, we don't need the requestedSlug anymore
-    const metadataParse = teamMetadataSchema.safeParse(prevOrganisation.metadata);
-    if (metadataParse.success) {
-      const { requestedSlug: _, ...cleanMetadata } = metadataParse.data || {};
+  if (input.slug) {
+    if (
+      IS_TEAM_BILLING_ENABLED &&
+      /** If the team doesn't have a slug we can assume that it hasn't been published yet. */
+      !prevOrganisation.slug
+    ) {
+      // Save it on the metadata so we can use it later
       data.metadata = {
-        ...cleanMetadata,
```

Comment:
This doesn't look right to me.

Reply:
I don't understand, that line was never there to begin with 🤔

Reply:
Maybe it's a GitHub glitch, because that line actually is in line 87, not after line 76. https://github.com/calcom/cal.com/blob/093ea1f35d1e2b922d332bc93b47e3275d6bb276/packages/trpc/server/routers/viewer/organizations/update.handler.ts#L87

Reply:
oh yeah it's line 77 - is that OK? because it is gone and loses all metadata except requestedSlug